bot.py

```diff
@@ -204,7 +223,7 @@ async def is_admin(context: commands.Context) -> bool:
     """
     global config
 
-    if str(context.author.id) not in config["admins"]:
+    if context.author.id not in config["admins"]:
```

Comment:
Ehhhhhhhhhhh can we get some real confirmation on what type the admin IDs in config should be?
When I set this up a bit earlier, I had to convert them all to strings

Reply:
Because you were reading strings from bot.conf. https://discordpy.readthedocs.io/en/latest/api.html#discord.User.id says they're ints.